### ISSUE TITLE
Clean bmatrix/Bmatrix/etc from latex output when generating output for matplotlib rendering

### ISSUE THREAD

**Issue by flacjacket** [sympy/sympy#3006]
Since there is a problem with the matplotlib rendering of latex matrices that include braces (such as bmatrix), in interactive/printing.py, the latex generation for matplotlib should include a workaround for this. This can be seen by running, from either a qtconsole or notebook:

```
In [1]: init_printing(use_latex='matplotlib')

In [2]: [x,y]
/home/sean/ipython/IPython/core/formatters.py:239: FormatterWarning: Exception in image/png formatter: 
\begin{bmatrix}x, & y\end{bmatrix}
^
Unknown symbol: \begin (at char 0), (line:1, col:1)
  FormatterWarning,
Out[2]: [x, y]
```

**Comment by moorepants** [comment]
This works fine for me on IPython 4 and SymPy master.

![selection_006](https://cloud.githubusercontent.com/assets/276007/10269126/f0f45312-6a81-11e5-84f7-2608da8249b6.png)

**Comment by moorepants** [comment]
I'm closing this. Please reopen if you can show how to reproduce.

**Comment by flacjacket** [comment]
Looks like this was fixed a while ago e5326881b16ac5c73dac95ef2fef6fbfb81fdf62, so this can stay closed
